.github/workflows/ci.yaml

```diff
@@ -24,7 +24,7 @@ on:
     branches:
       - master
 jobs:
-  run-tests:
+  code-tests:
```

Comment:
great idea to split up the jobs! if the deployment jobs has a flaky fail in the future, only the deployment job needs to be rerun right?

Reply:
I encountered this a couple of months before and I think its still a FR on Github side
Source: https://github.community/t5/GitHub-Actions/Ability-to-rerun-just-a-single-job-in-a-workflow/td-p/41629

Reply:
Yeah, I think you have to re-run it all. The code tests are super quick though, so re-running isn't a huge deal

Reply:
😭